packages/patternfly-4/package.json

```diff
@@ -30,5 +30,8 @@
     "lint:versions": "node ./verifyCoreVersions.js",
     "lint:styles": "yarn stylelint",
     "stylelint": "stylelint '**/sass/**/*.scss' !**/dist/**/*.scss"
+  },
+  "dependencies": {
+    "@types/jest": "^23.3.10"
```

Comment:
Also added in the root package. Can we move the types closer to react-core?

Reply:
Just tried removing these, but remember that these are needed because jest runs at the root and it can't find the @types causing the tests to fail.